src/coreclr/src/jit/emitxarch.cpp

```diff
@@ -2669,6 +2669,45 @@ void emitter::emitLoopAlign()
     emitCurIGsize += 15;
 }
 
+/*****************************************************************************
+ *
+ *  The next instruction will be a loop head entry point
+ *  So insert a dummy instruction here to ensure that
+ *  the x86 I-cache alignment rule is followed.
+ */
+
+void emitter::emitVariableLoopAlign(unsigned short alignmentBoundary)
+{
+    unsigned short nPaddingBytes    = alignmentBoundary - 1;
```

Comment:
Is it really worth making these `short` instead of `int`?

Reply:
I just felt that `unsigned short` type should be good enough to hold the values in these variables. Do you prefer using `unsigned` instead?